python/lsst/ts/criopy/m1m3/force_calculator.py

```diff
@@ -72,6 +73,87 @@ def reduce_to_y(forces: list[float]) -> Generator[float, None, None]:
             yield forces[fa.index]
 
 
+class ForceTable:
+    """
+    Combines table with comments stored in CSC file. Export methods allowing
+    loading and saving table/comments pair, and resetting and appending
+    comments.
+    """
+
+    def __init__(self, *args: Any, **kwargs: Any):
+        self.data = pd.DataFrame(*args, **kwargs)
+        self.comments: list[str] = []
+
+    def drop(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
The type annotation of `**kwargs` should be `dict` or something similar instead of `Any`.

Reply:
No, this is correct: https://stackoverflow.com/questions/37031928/type-annotations-for-args-and-kwargs